lib/l10n/app_de.arb

```diff
@@ -1,6 +1,24 @@
 {
     "@@locale": "de",
 
+    "price": "price: {amount}€",
+    "@price": {
```

Comment:
You don't need the templating in all files, only in app_en.arb because this is the default as set in the l10n.yaml

Reply:
okay thanks